db/migrate/20191017173513_create_highlights.rb

```diff
@@ -0,0 +1,24 @@
+# OpenStax Highlights
+class CreateHighlights < ActiveRecord::Migration[5.2]
+  def change
+    create_table :highlights do |t|
+      t.uuid :user_uuid, null: false, index: true
+      t.integer :source_type, null: false, default: 0
+      t.uuid :source_uuid, null: false
+      t.jsonb :source_metadata, null: true
+      t.text :source_parent_ids, array: true, default: []
```

Comment:
add index (`gin`) like in https://coderwall.com/p/sud9ja/rails-4-the-postgresql-array-data-type ?